lib/components/narrative/tabbed-itineraries.js

```diff
@@ -15,7 +15,7 @@ class TabbedItineraries extends Component {
   static propTypes = {
     itineraries: PropTypes.array,
     itineraryClass: PropTypes.func,
-    pending: PropTypes.number,
+    pending: PropTypes.bool,
```

Comment:
pending should be a number per batch routing changes.

Reply:
I fixed the wrong itineraries list... the correct one is taken care in https://github.com/opentripplanner/otp-react-redux/pull/242/commits/214e266d7b2b56c0e667358d9a5cac689a23212e.